lib/js2xml.js

```diff
@@ -42,7 +42,7 @@ function writeAttributes(attributes, options) {
     var key, result = '';
     for (key in attributes) {
         if (attributes.hasOwnProperty(key)) {
-            result += ' ' + key + '="' + writeText(attributes[key], {}) + '"';
+            result += ' ' + key + '="' + writeText(attributes[key], options) + '"';
```

Comment:
We can extract escaping function from writeText and use that function from here to be more consistent. What do you think?